src/dos/program_mixer.cpp

```diff
@@ -260,23 +801,29 @@ void MIXER::AddMessages()
 	        "  [color=light-green]mixer[reset] [/listmidi]\n"
```

Comment:
`[color=light-cyan][CHANNEL][reset]` suggests the channel is optional, where in fact is is required. It should be changed to `[color=light-cyan]CHANNEL[reset]`.

Reply:
It is optional, e.g., `MIXER x30 r10 c50` is entirely valid. See the unit tests for more examples.